src/main/java/utils/BrowserManagement.java

```diff
@@ -45,6 +52,7 @@ public static WebDriver initializeDriver() {
                 ChromeOptions chromeOptions = new ChromeOptions();
                 chromeOptions.addArguments("--start-maximized");
                 chromeOptions.addArguments("--incognito");
+                chromeOptions.setHeadless(true);
```

Comment:
We talked about this in private 👍🏽 This is fine for the purpose of this training